client/locales/en.yml

```diff
@@ -51,6 +51,7 @@ forms:
   login:
     errors:
       missing_username: Username is required
+      invalid_username: Invalid username format
```

Comment:
Needs a more detailed message for the restrictions (even though it will usually be impossible to happen)

Reply:
I was thinking the required format would only be shown on signup or user creation form... What do you think?

Reply:
you're right, better ux